dev/com.ibm.ws.jdbc_fat_krb5/publish/files/oracle/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM kyleaure/oracle-18.4.0-xe-prebuilt
+
+# Update and install Kerberos (server head)
+RUN yum update -y
+RUN yum -y install krb5-workstation krb5-libs krb5-auth-dialog vim
+
+COPY setup/ /opt/oracle/scripts/setup
+COPY startup/ /opt/oracle/scripts/startup
+
+ENV ORACLE_PWD=password
+ENV KRB5_REALM=EXAMPLE.COM
+ENV KRB5_KDC=kerberos
+
+RUN mkdir /etc/krb5
+RUN printf 'add_entry -password -p XE/oracle@EXAMPLE.COM -k 1 -e aes256-cts\npassword\nwkt /etc/krb5.keytab' | ktutil
```

Comment:
Wondering if this is necessary since this principle is no longer being used.

Reply:
To make thing a little bit clearer. We should have two keytab files,
1. keytab file for liberty server has the Kerberos principal TESTUSER@EXAMPLE.COM
2. keytab file for Oracle server has the XE/myhost@EXAMPLE.COM.
I thought we want to name the hostname to something else rather than oracle.
Also the Kerberos user principal is something like testuser or user in uppercase instead of oracle
... For example:
Liberty server's keytab has Kerberos principal testuser
the SPN is XE/myhost@EXAMPLE.COM